src/commands/checkout.js

```
@@ -718,12 +718,22 @@ async function updateWorkingDir(
       const batch = tasks.slice(i, i + batchSize).map(task => task())
       const batchResults = await Promise.allSettled(batch)
       batchResults.forEach(result => {
-        if (result.status === 'fulfilled') results.push(result.value)
+        if (result.status === 'fulfilled') {
+          results.push(result.value)
+        } else {
+          rejections.push(result.reason)
+          // eslint-disable-next-line no-console
+          console.error(
+            `[isomorphic-git ${operationName}] task rejected:`,
+            result.reason && result.reason.stack ? result.reason.stack : result.reason
```

ESLint

[error] Replace `·?·result.reason.stack` with `⏎··············?·result.reason.stack⏎·············`

(prettier/prettier)